ES/ESmy020.xml

```diff
@@ -36,305 +36,277 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
-                      <incipit>
-                      <q xml:lang="gez"><hi rend="rubric">ወንጌል፡ ዘማቴዎስ፡ እስመ፡ ከመ፡ <sic resp="DN">ከመ</sic>፡ ብእሲ፡</hi> ዘይነግድ፡
+                      <incipit xml:lang="gez">
+                        <locus target="#5ra"/>
+                      <hi rend="rubric">ወንጌል፡ ዘማቴዎስ፡ እስመ፡ ከመ፡ <sic resp="DN">ከመ</sic>፡ ብእሲ፡</hi> ዘይነግድ፡
                        ወጸውዓ፡ አግብርቲሁ፡ ወወሃቦሙ፡ ንዋየ፡ ይትገበር፨ ወቦ፡ ለዘወሀቦ፡ ፭፡ መክሊተ፡ ወቦ፡ ለዘወሀቦ፡ ፪፡ መክሊተ፡
-                        </q></incipit>
-                        <explicit>
-                        <q xml:lang="gez">ብካይ፡ ወሐቄ፡ ስነን፨
-                        </q></explicit>
+                      </incipit>
+                        <explicit xml:lang="gez">
+                          <locus target="#6ra"/>
+                        ብካይ፡ ወሐቄ፡ ስነን፨
+                        </explicit>
                     </msItem>
 
-                    <msItem xml:id="ms_il.6">
+                    <msItem xml:id="ms_i1.2">
                     <locus from="5ra" to="6vb"/>
-                    <title type="incomplete">Mǝlṭān and ʾaryām-antiphons for the priests</title>
-                    <incipit>
-                    <q xml:lang="gez"><hi rend="rubric">እሰግድ፡ ለከ፡ አብ፡ ቀዳማ</hi>ዊ፡ እሰግድ፡ ለከ፡ ወልድ፡ ዋህድ፡
+                    <locus from="2ra" to="2vb"/>
+                    <title type="complete">Mǝlṭān and ʾaryām-antiphons for the priests</title>
+                    <incipit xml:lang="gez">
+                      <locus target="#5ra"/>
+                    <hi rend="rubric">እሰግድ፡ ለከ፡ አብ፡ ቀዳማ</hi>ዊ፡ እሰግድ፡ ለከ፡ ወልድ፡ ዋህድ፡
                     ንሰግድለከ፡ ኦመንፈስ፡ ቅዱስ፨ ወሐሥሥ፡ በኃቤከ፡ በእንተ፡ ነፍስ፡ ገብርከ፡ እገለ፡ ዘወጽአ፡ እምዝንቱ፡ ሥጋ፡ ከመ፡ ኵሎሙ፡ አበዊነ፡
                     ለይኩን፡ ሎቱ፡ ዕረፍተ፡
-                      </q></incipit>
-                      <explicit>
-                      <q xml:lang="gez">ፍቁራኒሁ፡ ለብ፡ ኄራን፡ መተንበላ
-                      </q></explicit>
-                    <note>The end is <ref corresp="#ms_il.1"/>,
-                    cp. <foreign xml:lang="gez">ን፡ ኅቡረ፡ ይባርክዎ፡ ለአብ፨</foreign>.</note>
+                      </incipit>
+                    <explicit xml:lang="gez">
+                      <locus target="#2ra"/>
+                      ከመ፡ ትባኢ፡ ሃገረ፡ እንተ፡ ኬንያሃ፡ ወሐዋጺሃ> ወልደ፡ እግዚአብሔ<gap reason="lost"/>፨
+                    </explicit>
+                    <note>The text on <locus target="#6vb"/> ends with
+                    <foreign xml:lang="gez">ፍቁራኒሁ፡ ለብ፡ ኄራን፡ መተንበላ</foreign>.
+                  <locus target="#2"/> is in the reversed position, and the continuation is on the verso-side,
+                  <locus target="#2va"/>, cp. <foreign xml:lang="gez">ን፡ ኅቡረ፡ ይባርክዎ፡ ለአብ፨</foreign>.</note>
                   </msItem>
 
-                    <msItem xml:id="ms_il.7">
+                    <msItem xml:id="ms_i1.3">
                     <locus from="7ra" to="11rb"/>
                     <title type="complete" ref="LIT1931Mashaf#FuneralProcession">Directives and readings for the first two stations
```

Comment:
I would say the /title element ends after the first sentence, the rest is a note